README.md

```diff
@@ -75,7 +77,12 @@ the variable pizza inside your method.
 
 Your Answer:
 ```js
-// write code here
+// var food= pizza;
+  var temperature =70;
+  var method = bake;
+  function (method) {
+
+  }
```

Comment:
This starts out good.  It looks like you have to review adding methods to objects.